pkgs/racket-doc/scribblings/getting-started/getting-started.scrbl

```diff
@@ -1,5 +1,5 @@
 #lang scribble/doc
-@(require scribble/manual)
+@(require scribble/manual "../guide/guide-utils.rkt")
```

Comment:
This worries me, because I don't think of "getting-started" as building on the implementation of "guide", and someone modifying "guide-utils.rkt" probably would not expect changes to affect "getting-started" or other documents. Is there a better way to organization whatever needs to be shared?

Reply:
I pushed a change duplicating the definition of `Quick` from `guide-utils.rkt`.

There's another duplication of the `keep` function from `scribblings/quick/keep` in this commit; sharing it seemed like more trouble than needed here.